docs(perf-alert): functional description (#8414)
vatesfr/xen-orchestra: packages/xo-server-perf-alert/.USAGE.md

Like all xo-server plugins, it can be configured directly via
the web interface, see [the plugin documentation](https://xen-orchestra.com/docs/plugins.html).

> Monitors can be defined in the plugin configuration. These can monitor the storage usage of your SRs, the CPU usage or memory usage of your hosts and VMs.

Reviewer: ```suggestion
You can define monitors in the plugin configuration. Monitors let you check storage usage for your SRs, CPU usage or memory usage of your hosts and VMs.
```

> A separate alarm is created for each monitored element. This alarm will be raised when an element exceeds the defined threshold (on average during the last minute), and it will be lowered when it falls back below the threshold. An email alert is sent in both of those cases, and also if a problem is encountered when trying to access the stats of a monitored element.

Reviewer: ```suggestion
Each monitored element gets its own alarm. The alarm triggers when an element goes over the defined threshold (based on the average over the last minute) and clears when it drops back below. An email alert is sent when an alarm is raised and when it clears. If there's an issue accessing the stats for a monitored element, an email will be sent too.
```

> To ensure that each alert email is only sent once, the boolean state of each alarm is stored with an ID, e.g. `host|memoryUsage|40|803c2676-c309-721e-7123-e6c3de854c32` is the ID corresponding to the alarm that is raised when host `803c2676-c309-721e-7123-e6c3de854c32` exceeds 40% memory usage.

Reviewer: ```suggestion
To avoid sending duplicate alerts, each alarm's boolean state is stored with a unique ID. For example, `|host|memoryUsage|40|803c2676-c309-721e-7123-e6c3de854c32` represents an alarm triggered when host `803c2676-c309-721e-7123-e6c3de854c32` exceeds 40% memory usage.
```

> You can use the "Test plugin" option to receive an email containing the list of the monitors you have configured, and the status of the elements monitored.

Reviewer: ```suggestion
You can use the "Test plugin" option to get an email listing the monitors you've set up and the current status of each monitored element.
```
